src/dk/aau/cs/ds306e18/tournament/utility/configuration/RLBotConfig.java

```diff
@@ -110,4 +113,12 @@ public boolean setupMatch(Match match) {
         validateConfigSyntax();
         return isValid();
     }
+
+    /** Setup the parameters for a bot in the config file */
+    public void setupParticipant(Bot bot, int index, int team) {
+        setParticipantConfigPath(bot.getConfigPath(), index);
+        setParticipantTeam(team, index);
+        setParticipantType(bot.isPsyonixBot() ? BotType.PSYONIX : BotType.RLBOT, index);
```

Comment:
Since the RLBot-config allows for multiple types of players being handled by the system, e.g., `controller_passthrough` and `human`, hardcoding this type-setting is bad practice.